Answers.md

```diff
@@ -1,9 +1,14 @@
 1. If you had to describe semantic HTML to the next cohort of students, what would you say?
+it is the "meaning" of your html. It clearly describes its meaning to the browser and the developer. Pretty much it is how specific you are when writing html
 
 2. Describe some differences between ```display: block;``` and ```display: inline;```.
-
+block elements do not creat line breaks, they usualy take up the full width of the page
```

Comment:
I think you may have mixed this up - block elements do create line breaks while inline do not